.scripts/run_osx_build.sh

```diff
@@ -29,6 +29,9 @@ echo -e "\n\nSetting up the condarc and mangling the compiler."
 setup_conda_rc ./ ./recipe ./.ci_support/${CONFIG}.yaml
 mangle_compiler ./ ./recipe .ci_support/${CONFIG}.yaml
 
+echo -e "\n\nInstalling XQuartz using homebrew."
+brew install --cask xquartz
```

Comment:
This will be overwritten when we rerender the recipe. We need something [like this](https://github.com/conda-forge/conda-smithy/pull/539) to make these changes persist.

@isuruf sorry to bug you but I thought we had solved this. I could not find anything in conda-smithy or in our docs.

Reply:
You are right, and I, too, would like to see such a thing. But it might be worth noting that this is again about XQuartz, so perhaps [@jakirkham's comment](https://github.com/conda-forge/conda-smithy/pull/539#issuecomment-311515153) about finishing [the XQuartz package](https://github.com/conda-forge/staged-recipes/issues/2587) still applies?